packages/angular/tsconfig.ngc.json

```
@@ -0,0 +1,26 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+// This tsconfig is used when building @sentry/angular with the Angular
+// compiler and `ng-packagr`. It configures a production build conforming
+// to the Angular Package Format (APF).
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "target": "es2015",
+    "lib": [
+      "dom",
+      "es2018"
+    ],
+    "baseUrl": "./",
+  },
+  "angularCompilerOptions": {
+    "skipTemplateCodegen": true,
+    "strictMetadataEmit": true,
+    "enableResourceInlining": true,
+    // As per Angular 10, the recommendation from the library creation guide
+    // is to disable compilation for Ivy in production builds to ensure
+    // compatibility with Angular 10.
+    // For Angular 11-13 applications, ngcc and the Angular linker ensure that
+    // the library code is processed to match the specific Angular runtime version.
```

```suggestion
    // As per Angular 10, the recommendation from the library creation guide
    // is to disable compilation for the Ivy rendering engine in production builds
    // to ensure compatibility with Angular 10.
    // For Angular 11-13 applications, ngcc and the Angular linker ensure that
    // the library code is processed to match the specific Angular runtime version.
```

I'm not sure I quite understand this comment (specifically the second half). Does that mean that for 11-13 it will reenable ivy? Or will ignore this setting all together? (I also made a small edit to the first half just to give a non-angular reader a tiny bit of context here.)

(EDIT: Wow, do they make that hard to parse, with red on red. The change is to replace "Ivy" with "the Ivy rendering engine" and then move "to ensure" onto the next line for line wrapping purposes.)

---

I can see how this comment is confusing. Thanks for the suggestion, I included it and changed the second half of the explanation.

---

> Is this meant to point somewhere else, or is this copy pasta from somewhere else pointing to here?

Whoops, this should have linked to #4644. Changed it.

> I think this should now be `./build` rather than `./dist`, right?

Yes, fixed.

> Also, the repo uses yarn as our package manager/script runner, so probably more idiomatic to `s/npm run/yarn`.

Right, also fixed.


> I can't speak to the angular-specific bits, but overall this looks pretty reasonable.

Cool, thanks for the review!